docs/embedding/sdk/version.md

```diff
@@ -8,9 +8,9 @@ title: Embedded analytics SDK - versions
 
 {% include plans-blockquote.html feature="Embedded analytics SDK" %}
 
-The SDK version tracks with the Metabase version.
+The SDK stable version tracks with the Metabase version.
 
-So, for example, if you're on Metabase version `1.51.1`, you should use the SDK `1.51`.
+So, for example, if you're on Metabase version `1.51.2`, you should use the SDK `51-stable`.
```

Comment:
I think this is confusing because it makes it sounds like 51-stable is a version, when it is a dist-tag. I think both deserved to be introduced. Something like this.

```suggestion
So, for example, if you're on Metabase 51 (`0.51.x`, `1.51.x`), any version 1.51.x of the @metabase/embedding-sdk-react npm package will be compatible. To simplify things, we publish dist-tags for each stable Metabase version starting from Metabase 51,`51-stable`.

Use `npm install @metabase/embedding-sdk-react@51-stable` to install the latest version of the SDK compatible with Metabase 51.
To grab the latest version of the SDK that works with Metabase nightly builds, use the `canary` dist-tag.
```